src/app/(admin)/new_dashboard/shared/Card/index.tsx

```diff
@@ -11,6 +11,7 @@ type Props = {
   children?: JSX.Element;
   middlecontent?: string | JSX.Element;
   fullHeight?: boolean;
+  buttons?: JSX.Element;
```

Comment:
if it supposed to be a react element name it with a capital letter, and give more explanatory name